barretenberg/cpp/src/barretenberg/commitment_schemes/shplonk/shplonk.hpp

```diff
@@ -61,7 +62,20 @@ template <typename Curve> class ShplonkProver_ {
             Q.add_scaled(tmp, current_nu);
             current_nu *= nu;
         }
+        for (size_t idx = opening_claims.size(); idx < CONST_PROOF_SIZE_LOG_N + 2; idx++) {
```

Comment:
would be nice to add a comment explaining the significance of CONST_PROOF_SIZE_LOG_N + 2